observability/monitoring/monitoring-argo-cd-custom-resource-workloads.adoc

```diff
@@ -18,7 +18,7 @@ You can enable and disable the setting for monitoring Argo CD custom resource wo
 * You have installed the {gitops-title} Operator on your {OCP} cluster.
 * The monitoring stack is configured in your cluster in the `openshift-monitoring` project. In addition, the Argo CD instance is in a namespace that you can monitor through Prometheus.
 * The `kube-state-metrics` service is running on your cluster.
-* Optional: If you are enabling monitoring for an Argo CD instance already present in a user-defined project, ensure that the monitoring is link:https://docs.openshift.com/container-platform/latest/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[enabled for user-defined projects] in your cluster.
+* Optional: If you are enabling monitoring for an Argo CD instance already present in a user-defined project, ensure that the monitoring is link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[enabled for user-defined projects] in your cluster.
```

Comment:
```suggestion
* Optional: If you are enabling monitoring for an Argo CD instance already present in a user-defined project, ensure that the monitoring is link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[enabled for user-defined projects] in your cluster.
```

Reply:
@Dhruv-Soni11 please rectify the link in L37 too.

Reply:
Sure Sri. It is fixed now.

Reply:
@Dhruv-Soni11 L21 not fixed. Please change it.